src/Compute/Compute/help/New-AzCapacityReservationGroup.md

```diff
@@ -92,6 +92,21 @@ Accept pipeline input: True (ByPropertyName)
 Accept wildcard characters: False
 ```
 
+### -ProgressAction
+{{ Fill ProgressAction Description }}
```

Comment:
Why is this here?

Reply:
@Sandido This is a bug introduced by platyPS. We will take care of it.